scripts/qemu-check.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-SUPPORTED_PLATFORMS=(byt cht bdw hsw apl cnl sue icl)
+#SUPPORTED_PLATFORMS=(byt cht bdw hsw apl icl skl kbl cnl)
+# TODO: skl, kll, cnl is unstable need to check
```

Comment:
Typo kll/kbl
It's always good to also add TODOs/Fixes as issues, because it's easier to search there, for person that is just a user of feature.